exercises/hello-workflow/README.md

```diff
@@ -0,0 +1,74 @@
+# Exercise 1: Hello Workflow
+
+During this exercise, you will
+* Review the business logic of the provided Workflow Definition to understand its behavior
+* Modify the Worker initialization code to specify a task queue name
+* Run the Worker initialization code to start the Worker process
+* Execute the Workflow from the command line, specifying your name as input
+
+Make your changes to the code in the `practice` subdirectory (look for TODO 
+comments that will guide you to where you should make changes to the code). 
+If you need a hint or want to verify your changes, look at the complete version 
+in the `solution` subdirectory.
+
+## Part A: Review the Workflow Business Logic
+
+1. Open the `Workflow.cs` file (located in the `Worker` folder under `practice`) 
+   in the editor
+2. Review the input parameters, business logic, and return value. 
+
+## Part B: Specify a Task Queue Name for the Worker
+
+1. Open the `Program.cs` file (located in the `Client` folder under `practice`) 
+   in the editor
+2. Write `greeting-tasks` as the name of the task queue
+3. Save your changes
+4. Repeat and open the `Program.cs` file in the `Worker` folder under `practice)
+5. Write `greeting-tasks` as the name of the task queue and save
+
+
+## Part C: Start the Worker
+
+1. Make sure you're in the practice subdirectory. Run the following command in the terminal window to start the Worker
+
+```
+dotnet run --project Worker
+```
+
+Note: If you're running this on your local, make sure you are running the Temporal Service. In a new terminal window, run the following command: `temporal server start-dev`
```

Comment:
```suggestion
Note: If you're running this on your local environment, make sure you are running the Temporal Service. In a new terminal window, run the following command: `temporal server start-dev --ui-port 8080 --db-filename clusterdata.db`
```

Reply:
We could also just remove this all together because we have this instruction in the root README. In fact, I'd say we remove this all together so it stays consistent with other exercise READMEs which don't mention this part.